cluster/src/main/scala/io/snappydata/cluster/ExecutorInitiator.scala

```diff
@@ -167,10 +168,14 @@ object ExecutorInitiator extends Logging {
                     }
                     // TODO: Hemant: add executor specific properties from local
                     // TODO: conf to this conf that was received from driver.
-
-                    // If memory manager is not set, use Snappy unified memory manager
-                    driverConf.setIfMissing("spark.memory.manager",
-                      SNAPPY_MEMORY_MANAGER)
+                    if (Property.UseSparkMemoryManager.getOption(driverConf).exists(_.toBoolean)) {
```

Comment:
Instead of adding another property, won't it be better to add a token like "default" instead? So in SparkEnv where it checks for this property, it can do like:
```scala
conf.getOption("spark.memory.manager").filterNot(_.equalsIgnoreCase("default")).map(Utils.classForName(_)
...
```